app/javascript/src/components/Dashboard/Notes/Note/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Typography, Dropdown } from "neetoui";
+
+const Header = ({ title }) => (
+  <div className="flex justify-between font-bold ">
+    <Typography style="h4">{title}</Typography>
+    <Dropdown buttonStyle="text" icon={MenuVertical}>
+      <li>Edit</li>
+      <li>Delete</li>
```

Comment:
Go through the Slack thread.

Reply:
@chiragbagde Not helpful. If you were told the same sentence that you have written, that is `NeetoUI Menuitem.Button is giving an error.`, then would you be able to easily figure out what's the issue?

Put up a video showing what's the error and when it occurs. Also point out your neetoUI version.

Reply:
Sorry the message was vague and not properly communicated from my end. What I meant was when I try to render Dropdown from neetoui I was getting an error. MenuItem.Button from dropdown was giving this error. I have pointed out the issue in this video.
[https://www.loom.com/share/c4da54cb63cf40bfa50656c170072f1d](url)

Reply:
Initially I was also using docker. Later I tried bundle install but its giving me an error while installation similar to this video [https://www.loom.com/share/6378c61cfcef4bfdb92fcd63ef06cc41](url).  I tried the solution mentioned here but still its giving the same error.
<img width="653" alt="error-2" src="https://user-images.githubusercontent.com/47323008/191923556-3af2a167-62d5-4d01-ab88
<img width="654" alt="error-1" src="https://user-images.githubusercontent.com/47323008/191923573-eeb5322a-8490-449d-bfae-0208c5cf5563.png">
-55344ebafaea.png">

Reply:
Initially I was also using docker. Now when I try bundle install I'm getting an error while installation as 'Gem::Ext::BuildError: ERROR: Failed to build gem
native extension'.I tried the solution mentioned in slack video but it’s still giving the same error.
<img width="654" alt="error-1" src="https://user-images.githubusercontent.com/47323008/191929635-bd28b41f-f8e6-4ed9-84da-b30eef45d416.png">

<img width="653" alt="error-2" src="https://user-images.githubusercontent.com/47323008/191929671-c8ce6f4e-3c89-4c5e-a57b-91e76fb59926.png">

Reply:
>Initially I was also using docker. Now when I try bundle install I'm getting an error while installation as 'Gem::Ext::BuildError: ERROR: Failed to build gem
native extension'.I tried the solution mentioned in slack video but it’s still giving the same error. The error is occurring while installing ffi package.

Does this section resolve the issue: https://www.bigbinary.com/books/learn-rubyonrails-book/installing-ruby-on-rails#typeerror-for-rails-project-in-m1-mac ?

Reply:
>MenuItem.Button from dropdown was giving this error.

@chiragbagde Regarding the `MenuItem`. Refer the steps: https://github.com/Ishaan-Singhal/neetoUI-challenge/pull/9#discussion_r978600442

Also ensure you have restarted your Rails as well as webpack servers once you upgrade the versions.

Reply:
Now my server is running using bundle install and it also displays the MenuItem Button.